javascript/ql/src/experimental/Security/CWE-347-noVerification/jose.ql

```diff
@@ -0,0 +1,44 @@
+/**
+ * @name JWT missing secret or public key verification
+ * @description The application does not verify the JWT payload with a cryptographic secret or public key.
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 8.0
+ * @precision high
+ * @id js/jwt-missing-verification-jose
+ * @tags security
+ *       external/cwe/cwe-347
+ */
+
+import javascript
+import DataFlow::PathGraph
+
+DataFlow::Node unverifiedDecode() {
+  result = API::moduleImport("jose").getMember("decodeJwt").getParameter(0).asSink()
+}
```

Comment:
Is this just a second source/sink pair describing the same thing that is happening in `JsonWebToken.ql`.  
Why not combine the two into one `.ql` file?